ci/environment.yml

```diff
@@ -2,7 +2,6 @@ name: rubicon-dev
 channels:
   - conda-forge
 dependencies:
-  - python=3.8
```

Comment:
should we follow the strategy of offering different versions of the environment.yaml (`environment-3.7.yaml`, `environment-3.8.yaml`) and an `environment-latest.yaml` for ci? Or leave unpinned entirely?

Reply:
With the current setup, the version of Python in a given environment is defined by the actions matrix -- the benefit here being that we could add 3.9 support with just one edit, instead of adding an additional yaml file.
In terms of downstream users, their dependencies will be handled by `conda-forge` / `pip`.

Reply:
If we want it more explicit, though, that's definitely doable

Reply:
that makes sense -- I was thinking about when setting up a `rubicon-dev` environment locally

Reply:
Yeah -- instead of hiding that in a `ci` folder, maybe there can be a `environment-dev.yaml` file located in the repo root?